doc/source/docs/api/io.rst

```diff
@@ -0,0 +1,32 @@
+============
+Input/output
+============
+.. currentmodule:: geopandas
+
+GIS vector files
+----------------
+.. autosummary::
+   :toctree: api/
+
+   read_file
+
+PostGIS
+-------
+.. autosummary::
+   :toctree: api/
+
+   read_postgis
+
```

Comment:
I know you put only the top-level functions here, but it might also be useful to just list all IO together, so also the GeoDataFrame methods and not only strictly functions (without necessarily removing the methods from the GeoDataFrame page). So for example both `read_postgis` and `GeoDataFrame.to_postgis` here, so people can see them together.